packages/jspsych/src/JsPsych.ts

```diff
@@ -507,7 +527,34 @@ export class JsPsych {
     this.doTrial(this.timeline.trial());
   }
 
+  private getAutomaticMetaData() {
+
+    var variablesToBeUpdated = this.data.getVariablesToBeUpdated();
+
+    for (const variable of variablesToBeUpdated) {
+      var variableValues = this.data.get().select(variable).values;
+      var variableType = typeof variableValues.filter((v) => v !== null)[0];
+
+      var field = {
+        name: variable,
+        description: "Unknown",
+        value: variableType,
+        // If the variable type is string, then categories or levels are automatically calculated.
+        ...(variableType === 'string'  && {levels: this.data.getLevels(variable)}),
+        //Likewise if the variable type is numeric, min and max values are calculated.
+        ...(variableType === 'number' && {minValue: this.data.get().select(variable).min()}),
```

Comment:
Thinking about this a bit more, I'm not sure this works. The metadata file is supposed to have the min and max value for the whole dataset, but this would only get the min and max for a single participant.

Reply:
Got it! Removed.